### DISCUSSION TITLE
Premake5 Won't Install

### DISCUSSION THREAD

**Discussion by ro0ti** [premake/premake-core#1918] [Q&A]
After downloading the binary for my OS and attempting to launch it, it opens and closes immediately.

I've turned off all AV and attempted to relaunch even with an exclusion added, No luck. 

Anyone know how to fix?

**Comment by nickclark2016** [comment]
Premake is a standalone application, no install required. The binary that you are downloading is the application itself.  You the command line to execute the application.
